addons/binding/org.openhab.binding.pioneeravr/ESH-INF/thing/thing-types.xml

```diff
@@ -96,7 +129,7 @@
 		<state min="0" max="100" pattern="%d %%" />
 	</channel-type>
 
-	<channel-type id="volumeChannelDb" advanced="true">
```

Comment:
why did you remove the advanced flag here? Imho there should only be the dimmer channel by default - no need to have two volume controls for everyone.